BHoM_UI/Templates/Caller.cs

```diff
@@ -167,13 +205,15 @@ public virtual string Write()
         {
             try
             {
-                if (SelectedItem == null)
-                    return "";
-                else
-                    return SelectedItem.ToJson();
+                CustomObject component = new CustomObject();
```

Comment:
We now serialise also `InputParams` and `OutputParams`, since the user may have made some changes to the component that we want to keep track of.
Also, if the backend method does not exist anymore, the component does not reset to no inputs and no outputs.
This will help solve #91, to add the feature in https://github.com/BHoM/Grasshopper_Toolkit/issues/220